homeassistant/components/neato/camera.py

```diff
@@ -4,21 +4,30 @@
 
 from homeassistant.components.camera import Camera
 
-from . import NEATO_LOGIN, NEATO_MAP_DATA, NEATO_ROBOTS
+from .const import NEATO_DOMAIN, NEATO_MAP_DATA, NEATO_ROBOTS, NEATO_LOGIN
 
 _LOGGER = logging.getLogger(__name__)
 
 SCAN_INTERVAL = timedelta(minutes=10)
 
 
-def setup_platform(hass, config, add_entities, discovery_info=None):
+def async_setup_platform(hass, config, async_add_entities, discovery_info=None):
```

Comment:
Use `async def` instead of `def` to make these functions coroutines.

Reply:
Done